index.md

```diff
@@ -5,8 +5,12 @@ title: Hello, We Are Convivio
 
 Convivio is a boutique agency designing and developing digital services for innovative public and private sector organisations.
 
-That means we take clients beyond simply marketing themselves on the web, to delivering their core services online.
+That means we take clients beyond simply marketing themselves on the web, to delivering their core services online, using modern technology and processes to achieve great <br class="only-full-width" />user experiences.
 
-Our team has delivered this digital transformation before in the government, finance, and media sectors.
+We have delivered hugely successfully digital transformation before in the government, finance, and media sectors.
```

Comment:
typo - "hugely successfully" should be 'hugely successful"
However, I prefer not to use words like hugely, so could we just say successful?